app.ts

```diff
@@ -0,0 +1,414 @@
+// Get the container from the html where the data will be stored
+let recordNav: any = document.querySelector("#record-navigation-container"); //Navigation area;
```

Comment:
These do not need to be global variables. Global variables are generally frowned upon. There are a few acceptable global variables but try to remove these from being global.